src/rcc.rs

```diff
@@ -768,6 +768,19 @@ impl CFGR {
 
         while rcc.cfgr.read().sws().bits() != sysclk_src_bits {}
 
+        // Enable PLLSAI2
```

Comment:
Is this something we want to enable unconditionally? The mentions of PLLSAI2 are rather sparse in the `stm32l41xxx`/`42xxx`/`43xxx`/`44xxx`/ `45xxx`/`46xxx` reference manual.